openai-core/src/commonMain/kotlin/com.aallam.openai.api/run/RunRequest.kt

```diff
@@ -48,6 +51,45 @@ public data class RunRequest(
      * Keys can be a maximum of 64 characters long, and values can be a maximum of 512 characters long.
      */
     @SerialName("metadata") val metadata: Map<String, String>? = null,
+
+    /**
+     * What sampling temperature to use, between 0 and 2.
+     * Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic.
+     */
+    @SerialName("temperature") val temperature: Int? = null,
```

Comment:
```suggestion
    @SerialName("temperature") val temperature: Double? = null,
```
I believe this should be a floating number